docker/appserver/JBoss/src/configuration/standalone.xml

```diff
@@ -436,23 +519,64 @@
                 <smtp-server outbound-socket-binding-ref="mail-smtp"/>
             </mail-session>
         </subsystem>
+        <subsystem xmlns="urn:jboss:domain:messaging-activemq:4.0">
+           <server name="default">
+               <security elytron-domain="ApplicationDomain"/>
+               <statistics enabled="${wildfly.messaging-activemq.statistics-enabled:${wildfly.statistics-enabled:false}}"/>
```

Comment:
moet wildfly hier niet jboss zijn?

Reply:
Nee, de standalone-full.xml die in de container zit komt van JBoss zelf, en die gebruikt ook die regel.